src/System.Numerics.Tensors/src/System/Numerics/Tensors/ITensor.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Numerics.Tensors
+{
+    public interface ITensor<T>
+    {
+        ReadOnlySpan<int> Dimensions { get; }
+        bool IsReversedStride { get; }
```

Comment:
Maybe we name IsColumnMajor instead, even though its a 2d biased name.  Alternatively we just remove this since same info is in Strides.

Reply:
I agree "column major" or "row major" is 2d biased, but it succinctly describes the setting with little confusion.

Looking at http://www.ecma-international.org/publications/standards/Ecma-335.htm, it says:

> Array elements shall be laid out within the array object in row-major order (i.e., the elements
associated with the rightmost array dimension shall be laid out contiguously from lowest to
highest index). 

So even here, in the ECMA spec, it uses the term "row-major order", even when talking about multi-dimensional arrays. I think we would be OK to use the same term.

____________

One idea for an alternative: `IsCStyleOrder`/`IsOrderCStyle`